hw/hexagon/hexagon_dsp.c

```diff
@@ -138,9 +141,7 @@ static void hexagon_common_init(MachineState *machine, Rev_t rev,
                     NULL);
             sysbus_mmio_map(SYS_BUS_DEVICE(l2vic_dev), 1,
                 m_cfg->cfgtable.fastl2vic_base << 16);
-        }
-
-        if (!qdev_realize_and_unref(DEVICE(cpu), NULL, errp)) {
+        } else if (!qdev_realize_and_unref(DEVICE(cpu), NULL, errp)) {
```

Comment:
Oh is *that* what it objected to? `else-if` on the next line?

I must've misread the output, or I'd have fixed it myself.  Sorry.

Reply:
Ok this LGTM.